integration_tests/e2e/prisonerList/prisonerList.cy.ts

```diff
@@ -148,6 +148,40 @@ context(`Display the prisoner list screen`, () => {
         .firstRowLocationIs(expectedFirstRowInTable.location)
     })
   })
+
+  describe('pagination', () => {
+    it('should display pagination controls, displaying 1 to 10 with the next link, on the first page of the prisoner list', () => {
+      // Given
+      cy.signIn()
+      cy.visit('/')
+
+      // When
+      const prisonerListPage = Page.verifyOnPage(PrisonerListPage)
+
+      // Then
+      prisonerListPage //
+        .hasPaginationControlsDisplayed()
+        .hasNumberOfPagesDisplayed(10)
+        .hasNextLinkDisplayed()
+    })
+
+    it('should display pagination controls, displaying 2 to 11 with the next and previous links, on the 7th page of the prisoner list', () => {
+      // Given
+      cy.signIn()
+      cy.visit('/')
+
+      // When
+      const prisonerListPage = Page.verifyOnPage(PrisonerListPage)
+
+      // Then
+      prisonerListPage //
+        .hasPaginationControlsDisplayed()
+        .hasNumberOfPagesDisplayed(10)
+        .setPaginationNumber(7)
+        .hasPreviousLinkDisplayed()
+        .hasNextLinkDisplayed()
+    })
+  })
```

Comment:
I'm not sure this is what I had in mind .... I think I was thinking more something like this (most of it is pseudo code)
```
    it('should display pagination controls, displaying 2 to 11 with the next and previous links, on the 7th page of the prisoner list', () => {
      // Given
      cy.signIn()
      cy.visit('/')
      const prisonerListPage = Page.verifyOnPage(PrisonerListPage)

      // When
      prisonerListPage.gotoPage(7) // I think "gotoPage" is a better name than your current "setPaginationNumber" method, but its basically the same thing

      // Then
      prisonerListPage //
        .doesNotHavePaginationLinkForPage(1)
        .hasPreviousLinkDisplayed()
        .hasPaginationLinkForPage(2)
        .hasPaginationLinkForPage(3)
        .hasPaginationLinkForPage(4)
        .hasPaginationLinkForPage(5)
        .hasPaginationLinkForPage(6)
        .paginationCurrentPageIs(7)
        .hasPaginationLinkForPage(8)
        .hasPaginationLinkForPage(9)
        .hasPaginationLinkForPage(10)
        .hasPaginationLinkForPage(11)
        .hasPaginationLinkForPage(12)
        .hasNextLinkDisplayed()
        .doesNotHavePaginationLinkForPage(13)
    })
```
Something like that perhaps?
There might be better ways to express that we are expecting a specific set of pagination controls,  but you get the idea

Reply:
Will revisit!